pages/requests/[id].js

```diff
@@ -34,6 +34,8 @@ const Request = () => {
   const isLoading = isLoadingRequest || isLoadingSOWs || isLoadingMessages
   const isError = isRequestError || isSOWError || isMessagesError
 
+  if (isLoading) return <Loading wrapperClass='item-page mt-5' />
```

Comment:
moved our loading state to come before checking for a user so that the page doesn't glitch for a signed in user.